setup.cfg

```diff
@@ -69,12 +69,11 @@ install_requires =
     jinja2==3.0.*
     metomi-isodatetime>=1!3.0.0,<1!3.2.0
     # Constrain protobuf version for compatible Scheduler-UIS comms across hosts
+    packaging
     protobuf>=4.21.2,<4.22.0
     psutil>=5.6.0
     pyzmq>=22
-    # https://github.com/pypa/setuptools/issues/3802
-    setuptools>=49,!=67.*
```

Comment:
Are we sure we want to remove this dependency on setuptools entirely? We could make it an optional dependency at least to ensure we don't get v67?

Reply:
This is the "install-requires" section which specifies Cylc's runtime dependencies, not its build dependencies. With this change, setuptools is no longer a runtime dependency.

Reply:
Ok, can't even reproduce the setuptools 67 problem anymore anyway